manifest.json

```diff
@@ -1,28 +1,22 @@
 {
-    "manifest_version": 2,
```

Comment:
Sorry about the huge diff, looks like my code formatter ran. I'll try to remove the formatting changes to clear up the diff.

Reply:
Should be fixed now